### DISCUSSION TITLE
Integration with CodeMirror or Monaco

### DISCUSSION THREAD

**Discussion by FlatMapIO** [facebook/lexical#4084] [General]
Code Editor is a relatively complex sub-Editor, Lexical as an Editor will always have problems integrating with CodeMirror & Monaco, event conflicts cause strange problems and are difficult to troubleshoot, operation history is not uniform, and yjs bindings are challenging in many ways, I would like to provide a mechanism to integrate Lexical with these powerful Code Editors, any suggestions?

**Comment by pplonski** [comment]
I would be interested in such intergration 👀
